### ISSUE TITLE
Missing translation term

### ISSUE THREAD

**Issue by trk** [AppoloDev/bolt-seo#9]
Tranlation term on : https://github.com/AppoloDev/bolt-seo/blob/main/src/Seo/Seo.php#L127 missing

**Comment by Fredxd** [comment]
Hi,
Thanks for feedback, will try to look this week if i can.

**Comment by trk** [comment]
We can only display content type name, or you can give an option to user, for example i don't want a automatically added text to title like `Listing of %contenttype%`

```php
case 'listing':
    return $this->cleanUp(
        $this->contentType->get('name').$this->postfixTitle()
    );
```

**Comment by Fredxd** [comment]
Hi,
Sounds good for me.
Can you create a PR for this please ?

**Comment by Fredxd** [comment]
Merged in #12 into develop branch
Thanks @trk
